### DISCUSSION TITLE
Communication among pods encrypted by TLS and 24-hour certificate rotation evidence?

### DISCUSSION THREAD

**Discussion by visdubey** [linkerd/linkerd2#5265] [Q&A] [answered]
I have been asked below question by the team, Could you please help me to provide reference/pointer to get those screens/evidence? 
Would you be able to provide evidence that communication among pods within the Kubernetes is automatically encrypted using mutual TLS provided by LinkerD?  I would like to see that it’s TLS 1.2.  
Additionally, is there any type of evidence to support the 24-hour certificate rotation?

**Comment by grampelberg** [comment, marked as answer]
Look at the [securing your service](https://linkerd.io/2/tasks/securing-your-service/) docs to find verification steps.

Look at the certificate after 24 hours to see if it is different.

**Reply by visdubey** [reply]
Can you please provide more details to it like how to check using commands or any reference in yaml manifest file? I am not able to pinpoint it using your provided answer.

**Reply by grampelberg** [reply]
The doc includes commands, please follow those commands.
